api/utils/keypaths/keypaths.go

```diff
@@ -111,13 +111,16 @@ const (
 //    │   ├── foo-kube                 --> Kubernetes certs for user "foo"
 //    │   |    ├── root                 --> Kubernetes certs for Teleport cluster "root"
 //    │   |    │   ├── kubeA-kubeconfig --> standalone kubeconfig for Kubernetes cluster "kubeA"
-//    │   |    │   ├── kubeA-x509.pem   --> TLS cert for Kubernetes cluster "kubeA"
+//    │   |    │   ├── kubeA.crt        --> TLS cert for Kubernetes cluster "kubeA"
```

Comment:
RFD136 described a bit different naming scheme, did we change it?

Reply:
Yes, I'm changing it in this PR https://github.com/gravitational/teleport/pull/44609